cmd/state/internal/cmdtree/cmdtree.go

```diff
@@ -137,6 +137,55 @@ func newStateCommand(globals *globalOptions, prime *primer.Values) *captain.Comm
 		"",
 		locale.T("state_description"),
 		prime.Output(),
+		[]captain.CommandGroup{
+			{
+				Message:  locale.Tl("packages_group_message", "Package Management"),
+				Commands: []string{"packages"},
+			},
+			{
+				Message: locale.Tl("environment_group_message", "Environment Management"),
+				Commands: []string{
+					"activate",
+					"init",
+					"clean",
+					"deploy",
+				},
+			},
+			{
+				Message: locale.Tl("platform_group_messages", "Platform Tools"),
+				Commands: []string{
+					"auth",
+					"invite",
+					"organizations",
+					"platforms",
+					"projects",
+					"show",
+				},
+			},
+			{
+				Message: locale.Tl("automation_group_messages", "Automation"),
+				Commands: []string{
+					"events",
+					"scripts",
+				},
+			},
+			{
+				Message: locale.Tl("version_control_group_messages", "Version Control"),
+				Commands: []string{
+					"fork",
+					"pull",
+					"push",
+					"history",
+				},
+			},
+			{
+				Message: locale.Tl("utilities_group_message", "Utilities"),
+				Commands: []string{
+					"update",
+					"export",
+				},
+			},
+		},
```

Comment:
I realize this strays a bit from what we discussed. However it feels in line with how we create our new commands. Additionally this does not require us to keep a mapping of group names to commands. We can search for the commands as we encounter them in the template.